src/PDB_Types.h

```diff
@@ -14,10 +14,12 @@ namespace PDB
 	// emulating std::byte from C++17 to make the intention clear that we're dealing with untyped data in certain cases, without actually requiring C++17
 	enum class Byte : unsigned char {};
 
+	typedef void* HANDLE;
+
 	// this matches the definition in guiddef.h, but we don't want to pull that in
 	struct GUID

```

Comment:
I think a better solution would be to use the uint*_t types here. This is less guesswork for the reader as to how large a struct is, and should make it more compatible with other platforms & compilers.